hw2/04-background.js

```diff
@@ -1 +1,63 @@
 // Add your code here
+const elementToStyle = document.querySelector('body');
+const input = document.querySelector('input');
+const buttonInput = document.querySelector('button');
+let intervalTime = 0;
+let interval = null;
+let randomColorEnabled = true;
+
+input.addEventListener('input', handleInput);
+buttonInput.addEventListener('click', handleButtonPress);
+window.addEventListener('load', handelOnLoad());
+
+function handleButtonPress(event) {
+  event.preventDefault();
+  if (randomColorEnabled) {
+    clearInterval(interval);
+    randomColorEnabled = false;
+    buttonInput.innerText = 'Start';
+    buttonInput.classList.remove('btn-danger');
+    buttonInput.classList.add('btn-primary');
+  } else {
+    startInterval();
+    randomColorEnabled = true;
+    buttonInput.innerText = 'Stop';
+    buttonInput.classList.remove('btn-primary');
+    buttonInput.classList.add('btn-danger');
+  }
+}
+
+function handleInput(event) {
```

Comment:
User should only be able to change the interval when the color changing has stopped. This in unnecessary.

"You should implement a button that allows the user to “stop” and “start” the background color change. Once the background color change is stopped, the user should be able to specify a new interval for changing the background color (in seconds)"